packages/terra-heading/src/terra-dev-site/doc/example/HeadingColors.jsx

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
 import Heading from 'terra-heading/lib/Heading';
+import classNames from 'classnames/bind';
+import styles from './colors.module.scss';
+
+const cx = classNames.bind(styles);
 
 const HeadingColors = () => (
   <div>
-    <Heading level={3} color="Crimson">Font Color: Crimson</Heading>
-    <Heading level={3} color="rgb(46,125,50)">Font Color: rgb(46,125,50)</Heading>
-    <Heading level={3} color="#304FFE">Font Color: #304FFE</Heading>
+    <Heading level={3} className={cx(['attention'])}>Font Color: Crimson</Heading>
```

Comment:
```suggestion
    <Heading level={3} className={cx(['attention'])}>Font Color: 'attention' className</Heading>
```
nit - we're letting the end colors bleed into the example instead of the class implementation

Reply:
Fixed in https://github.com/cerner/terra-core/pull/2211/commits/618ae6178edc4a734ef31a044c3bc8d570051b5c